L3/app/Domain/Players/Actions/GetAllPlayersAction.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Domain\Players\Actions;
+
+use App\Domain\Players\Models\Player;
+
+class GetAllPlayersAction
+{
+    public function execute(): array
+    {
+        return Player::all()->toArray();
```

Comment:
Нужно возвращать коллекцию, а не массив. Иначе потом не возможно использовать ресурсы для ответов в контроллере. Т.е. должно быть просто
```suggestion
        return Player::all();
```